src/pages/PhoneSignIn.vue

```diff
@@ -89,6 +95,32 @@ export default {
       if (num_match != null) return true;
       return false;
     },
+
+    async handleClickGoogleSignIn(){
+      try {
+        const googleUser = await this.$gAuth.signIn();
+        if (!googleUser) {
+          return null;
+        }
+        this.user = googleUser.getBasicProfile().getEmail();
+        let socialAuthToken = googleUser.getAuthResponse();
+        await this.convertSocialAuthToken(socialAuthToken.access_token);
+        this.$router.push({name: 'Home'});

```

Comment:
this may not be fully correct, right? Similar to how we are doing in the `storePhone` method - we need to redirect back to the page they came from - how can we do that here?

Reply:
Hmm... while have to think about it. Not sure what's the best way in vue right now.

Reply:
I think we can target it separately. I've created a separate tracking here:
https://github.com/avantifellows/plio-frontend/issues/144